rhel7/overlays/stig_overlay.xml

```diff
@@ -1,991 +1,2 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did the overlay become a non-null terminated one liner? :laughing:

Reply:
Something went wrong with the `create-stig-overlay.py` utility. Let me check.

Reply:
I guess there is something going on with the ElementTree python class and library versions of `xml.etree`. It's not pretty printing the XML.

Reply:
I couldn't make it work with `xml.etree` module [from here](https://github.com/ComplianceAsCode/content/blob/master/ssg/xml.py#L10). I had to use `lxml` combined with `pretty_print=True` when writing the contents to the file.

Reply:
Do you think it makes sense to update the generating code?

Reply:
I've opened: https://github.com/ComplianceAsCode/content/pull/6105 please check it out